projects/plugins/boost/app/contracts/Has_Slug.php

```diff
@@ -2,5 +2,5 @@
 namespace Automattic\Jetpack_Boost\Contracts;
 
 interface Has_Slug {
-	public function get_slug();
+	public static function get_slug();
```

Comment:
This is now a static method which we are calling via an instance object elsewhere in the code. e.g.: [this call](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/boost/app/features/optimizations/Optimization.php#L21).

I was worried about this at first, but read that [it is a supported feature with no plans for deprecation](https://www.reddit.com/r/PHP/comments/3yph4q/is_it_bad_practice_to_call_a_static_method_via_an/cyg3fct/).

So I'm ok with this, but wanted to document why.

As a side-question, which syntax do you think is clearer when calling a static method via an instance object; `$instance->method();` or `$instance::method();`?

Both work, but I can't find documentation supporting the latter as having "official" support.

Reply:
I think `$instance::method();` is clearer.